composer.json

```diff
@@ -15,9 +15,9 @@
         }
     ],
     "require": {
-        "php": "^7.3",
+        "php": "^7.3||^8.0",
```

Comment:
Won't we need 8.1 too?

Reply:
Yes it should be included as part of [^8.0](https://getcomposer.org/doc/articles/versions.md#caret-version-range-), learned that recently :D